SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -34,31 +34,32 @@ void USpatialPackageMapClient::Init(USpatialNetDriver* NetDriver, FTimerManager*
 void GetSubobjects(UObject* ParentObject, TArray<UObject*>& InSubobjects)
 {
 	InSubobjects.Empty();
-	ForEachObjectWithOuter(ParentObject, [&InSubobjects](UObject* Object)
-	{
-		// Objects can only be allocated NetGUIDs if this is true.
-		if (Object->IsSupportedForNetworking() && !Object->IsPendingKill() && !Object->IsEditorOnly())
-		{
-			// Walk up the outer chain and ensure that no object is PendingKill. This is required because although
-			// EInternalObjectFlags::PendingKill prevents objects that are PendingKill themselves from getting added
-			// to the list, it'll still add children of PendingKill objects. This then causes an assertion within
-			// FNetGUIDCache::RegisterNetGUID_Server where it again iterates up the object's owner chain, assigning
-			// ids and ensuring that no object is set to PendingKill in the process.
-			UObject* Outer = Object->GetOuter();
-			while (Outer != nullptr)
+	ForEachObjectWithOuter(
```

Comment:
These new lines seem odd - can't find a rule for it - what is it?

Reply:
Me neither, I think it relates to that fact that not all args can fit on one line, so instead we prefer to indent them 1 tab in all on separate lines, personally don't think it's too bad since it only occurs for such huge function calls.